.github/workflows/reusable-build-framework-flavor.yaml

```diff
@@ -48,6 +51,6 @@ jobs:
                 insecure = true
                 http = true
           EOF
-          earthly +build-framework-image --FLAVOR=${FLAVOR} --VERSION=master
+          earthly +build-framework-image --FRAMEWORK_FLAVOR=${FLAVOR} --VERSION=master
```

Comment:
ditto

Reply:
This one is a "reusable" workflow. It could build the fips or the generic one. If we know we'll never build the fips framework image ever (or any other, other than the generic), we can remove all the "reusable" logic from everywhere and just have one workflow building the generic image.